src/node.js

```diff
@@ -388,6 +393,58 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    function emitPendingUnhandledRejections() {
+      var hadListeners = false;
+      while (pendingUnhandledRejections.length > 0) {
+        var promise = pendingUnhandledRejections.shift();
+        var reason = pendingUnhandledRejections.shift();
+        if (hasBeenNotifiedProperty.get(promise) === false) {
+          hasBeenNotifiedProperty.set(promise, true);
+          if (!process.emit('unhandledRejection', reason, promise)) {
```

Comment:
@golyshevd I've actually since opened an issue about this where you are welcome to argue your stance: https://github.com/iojs/io.js/issues/830